services/data-info/src/data_info/routes/stats.clj

```diff
@@ -9,6 +9,8 @@
 
 (defroutes* stat-gatherer
 
+            ; FIXME Update metadactyl exception handling when data-info excptn hndlg updated
+            ; metadactyl catches exceptions thrown from this EP.
```

Comment:
Sorry, I wasn't referring to exceptions thrown in Donkey in general. I thought we were just talking specifically about errors returned from data-info in Donkey pass-through endpoints.